assets/differentialabundance_report.Rmd

```diff
@@ -608,8 +608,7 @@ for (assay_type in rev(names(assay_data))){
       cor_method = params$exploratory_cor_method,
       plot_title = paste0(
         paste0(params$observations_type," clustering dendrogram, "), 
-        params$exploratory_n_features, 
-        " most variable ", 
+        ifelse(params$exploratory_n_features == -1, "all ", paste0(params$exploratory_n_features, " most variable ")),
```

Comment:
Use the actual number of rows

Reply:
I can do that, but I'll also leave the word 'all' so that readers know that this number corresponds to all genes (i.e. ' based on all 1312 genes')